templates/default/datadog.conf.erb

```diff
@@ -183,6 +183,11 @@ end
 apm_enabled: <%= node['datadog']['enable_trace_agent'] %>
 <% end -%>
 
+[trace.config]
+<% unless node['datadog']['trace_env'].nil? -%>
+env = <%= node['datadog']['trace_env'] %>
```

Comment:
Could you use `env: ` instead of `env = ` here? I know the 2 syntaxes _should_ be similar in the `ini` format but still I'd rather be consistent with the other options